ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/ImagesTable.tsx

```diff
@@ -182,19 +182,19 @@ function ImagesTable({
                                     <DateDistanceTd date={scanTime} />
                                 </Td>
                                 <Td isActionCell>
-                                    <ActionsColumn
-                                        items={[
-                                            {
-                                                title: 'Watch image',
-                                                onClick: () =>
-                                                    onWatchImage(
-                                                        name
-                                                            ? `${name.registry}/${name.remote}:${name.tag}`
-                                                            : ''
-                                                    ),
-                                            },
-                                        ]}
-                                    />
+                                    {name?.tag && (
+                                        <ActionsColumn
+                                            items={[
+                                                {
+                                                    title: 'Watch image',
+                                                    onClick: () =>
+                                                        onWatchImage(
+                                                            `${name.registry}/${name.remote}:${name.tag}`
+                                                        ),
```

Comment:
Should there be an "Unwatch" option, if the image is already watched?

Reply:
Ah, yeah this is a great idea. Without this option the only way to unwatch an image is to open the modal and scan the list, manually matching up strings in the table. At the very least this shouldn't say "watch image" if the image is already watched.

@mansursyed Any thoughts about the flow here? Should an "unwatch" option pop open a different, simpler confirmation modal?

Reply:
Discussion moved to Slack, will likely implement an "Unwatch" option as a follow up.